### DISCUSSION TITLE
Everybody loves data except nobody loves data

### DISCUSSION THREAD

**Discussion by annyrviloria** [albertkun/21S-ASIAAM-191A#27] [Past Thinking Caps]
Well, it depends on what we mean by "we". I would argue that as a society, we simply do not use data. Generally speaking, policy makers are more easily persuaded by $lobbying$ than by evidence and facts. At the educational level there is no standard for instilling critical data skills even yet we are surrounded by corporations adept at massaging data and exploit our predilection for simple-yet-catchy promises. I think most people would say they use GIS regularly (google/apple maps) but again, not that many examples come to mind when it comes to the active use of geospatial data in politics or simply decision-making. In my own professional work in the public health field there are plenty of data collected by usually these data are hidden behind some institutional paywall or held secretively due to the strings that come attached to any grant these days. All in all, my sense is that data are all around us but that does not mean that data are used or embedded in the fabric of our society.

**Comment by Wny-Duong** [comment]
Critical data skills are definitely something that should be taught to the greater population, especially when they're often exposed to biased visualizations and sometimes manipulations of data that are crafted to promote some particular interest. However, I feel that I disagree on the idea that geospatial data isn't often used in policy and decision-making, as I've been taught that the technology is pretty essential when it comes to dealing with humanitarian crisis and assistance in matters such as addressing food insecurity, providing shelter and sanitation stations during natural disasters, and peacekeeping efforts in fragile states. 

I think the matter kind of draws back to the idea of critical data skills. I'm assuming the most of us in the course are American citizens, residents in the US, or have the benefit of living somewhere where the issues that GIS are used resolve don't appear very common. A better understanding of geospatial data, its contents, and how it can be used would allow people to take that kind of data into our political system to promote meaningful and recognizable causes, as issues regarding food insecurity and housing are current problems in the country that can't be adequately addressed without the geospatial data and sufficient understanding of how to use it.

**Comment by ninalong** [comment]
I think you make a really strong point about how data have become almost a resource for the privileged. Corporations and those educated to understand data are able to manipulate it to exert greater societal control, while those without access to data literacy are at their whim.
